numba/tests/test_np_functions.py

```diff
@@ -441,6 +440,18 @@ def unwrap(p, discont=None, axis=-1, period=6.283185307179586):
     return np.unwrap(p, discont, axis, period=period)
 
 
+def unwrap1(p):
```

Comment:
Just out of curiosity, is there any reasoning for the function naming convention?
* unwrap1
* unwrap13
* unwrap123

Reply:
This naming convention helps me keep track of which of `unwrap`'s arguments it needs as input(only first, first and third, etc.).